src/Camp24Devon/Tickets.elm

```diff
@@ -0,0 +1,238 @@
+module Camp24Devon.Tickets exposing (..)
+
+import AssocList
+import Camp24Devon.Product as Product
+import Element exposing (..)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font
+import Element.Input as Input
+import Env
+import Id exposing (Id)
+import MarkdownThemed
+import Money
+import PurchaseForm exposing (Accommodation(..), PurchaseForm)
+import Stripe exposing (Price, ProductId(..))
+import Theme
+
+
+
+-- @TODO need to use pricing IDs here, not product IDs
+-- but how do we figure out the right price given the current user? Is there a Stripe API for that?
+
+
+type alias Ticket =
+    -- @TODO change this
+    { name : String
+    , description : String
+    , image : String
+    , slots : Int
+    , productId : String
+    }
+
+
+attendanceTicket : Ticket
+attendanceTicket =
+    { name = "Campfire Ticket"
+    , description = "Attendee ticket for one person. Full access to the event 18th - 21st June, breakfast, lunch, tea & dinner included as per schedule"
+    , image = ""
+    , slots = 1
+    , productId = Product.ticket.attendanceTicket
+    }
+
+
+offsite : Ticket
+offsite =
+    { name = "Offsite"
+    , description = "You'll be organising your own accommodation off-site and making your own way to/from the event each day. You'll still have full access to the event and all meals."
+    , image = ""
+    , slots = 2
+    , productId = Product.ticket.offsite
+    }
+
+
+campingSpot : Ticket
+campingSpot =
+    { name = "Camping Spot"
+    , description = "Bring your own tent or campervan and stay on site. Showers & toilets provided."
+    , image = ""
+    , slots = 2
+    , productId = Product.ticket.campingSpot
+    }
+
+
+singleRoom : Ticket
+singleRoom =
+    { name = "Single Room"
+    , description = "Private room for a single attendee for 3 nights."
+    , image = ""
+    , slots = 2
+    , productId = Product.ticket.singleRoom
+    }
+
+
+doubleRoom : Ticket
+doubleRoom =
+    { name = "Double Room"
```

Comment:
Do we have a way for them to indicate which bed configuration they need?

Reply:
No not currently. Do we need to do that? I.e. are there some rooms that are not reconfigurable – i.e. it _must_ be occupied by a couple? Or could we just do it on the fly at the event?

Reply:
There are some unconfigurable rooms. A lot in fact. I think about 1/2. We might be able to 'guess' from what we know and allocate the set doubles to couples we know then the rest can be flexi. 
Probably don't want to make things more complicated than necessary.